app/helpers/number_helper.rb

```diff
@@ -69,8 +69,6 @@ def mhz_to_human_size(size, *args)
     nil
   end
 
-  private
-
```

Comment:
I tend to believe that `private` was used here and below to actually make the methods private but failed to do so because you can't make class methods private that way.  I'm fine with denoting a `private` section, perhaps with a comment but not with something that people might believe actually makes the methods below private.